DevCycle.SDK.Server.Cloud/Api/DVCCloudClient.cs

```diff
@@ -126,7 +126,7 @@ public async Task<Variable<T>> VariableAsync<T>(User user, string key, T default
             if (options.EnableEdgeDB) queryParams.Add("enableEdgeDB", "true");
 
 
-            return await GetResponseAsync<Dictionary<string, Variable<object>>>(user, urlFragment, queryParams);
+            return await GetResponseAsync<Dictionary<string, Dictionary<string, object>>>(user, urlFragment, queryParams);
```

Comment:
if you make that class described above it would also be used here to describe to the JSON deserializer how to parse the response.

Also local and cloud should use the same one (don't be fooled by there being a local Variable and cloud Variable class)